frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1509,488 +1779,245 @@ function ReaderFooter:addToMainMenu(menu_items)
                 },
             },
             {
-                text_func = function()
-                    return T(_("Progress percentage format: %1"),
-                        self:progressPercentage(tonumber(self.settings.progress_pct_format)))
-                end,
+                text = _("Max pct. of screen width used for text items"),
```

Comment:
The current string is better, discussion in https://github.com/koreader/koreader/pull/11678#discussion_r1569186152
```suggestion
                text = _("Maximum width of items"),
```